src/themes/oneui/base-variables/_misc.scss

```diff
@@ -1,5 +1,14 @@
 $transition-duration: .3s;
 
+/**
+ * Layering of components by z-index guide
+ * [0-100] — regular components
```

Comment:
maybe make this and the next range bigger. It is general practice to use steps of 10 for these things, so that it is easy to add things in between in the future, for that reason larger ranges are useful. Could even move the overall layover up to 700, as there are not much coming after that I suppose